src/eventtimes.jl

```diff
@@ -39,3 +39,9 @@ end
 
 isevent(ev::EventTime) = ev.status
 iscensored(ev::EventTime) = !ev.status
+
+## StatsModels compatibility
+
+StatsModels.concrete_term(t::Term, xs::AbstractVector{<:EventTime}, ::Nothing) =
+    StatsModels.ContinuousTerm(t.sym, first(xs), first(xs), first(xs), first(xs))
+Base.copy(et::EventTime) = et
```

Comment:
Is this actually necessary? Coverage says this line isn't covered which means it isn't being used by anything. (Anything that's currently tested, anyway.)

Reply:
huh, it should be hit by https://github.com/JuliaStats/StatsModels.jl/blob/master/src/terms.jl#L475

I'll look into it...

Reply:
Maybe it gets compiled away?  I was going to say it's probably because `EventTime` is immutable but I'm not sure.  When that line is removed there's an error because `modelcols` calls `copy.(table[term.sym])`

Reply:
Huh, weird.